modules/page_base.py

```diff
@@ -586,15 +586,15 @@ def hide_popup_by_class(self, class_name: str) -> None:
                 """
         self.driver.execute_script(script)
 
-    def hide_popup_by_child_node(self, node: WebElement, chrome=False) -> Page:
+    def hide_popup_by_child_node(
+        self, reference: Union[str, tuple, WebElement], labels=[]
+    ) -> Page:
+        node = self.fetch(reference, labels=labels)
         script = """var element = arguments[0].parentNode;
                  if (element && element.hidePopup) {
                     element.hidePopup();
                  }"""
-        if chrome:
-            with self.driver.context(self.driver.CONTEXT_CHROME):
-                self.driver.execute_script(script, node)
-        else:
+        with self.driver.context(self.context_id):
```

Comment:
Is there ever going to be a case where we have to dismiss a context menu in content context? I was just wondering bc of the change here

Reply:
I don't believe it's even possible to spawn a context menu in the content context.